go-controller/pkg/ovn/policy.go

```diff
@@ -168,18 +168,27 @@ func (oc *Controller) addIPBlockACLDeny(np *namespacePolicy,
 	return
 }
 
-func (oc *Controller) addACLDenyPortGroup(portGroupUUID, portGroupName,
-	priority string, policyType knet.PolicyType) error {
-	var match, direction string
-	direction = toLport
+func (oc *Controller) addACLPortGroup(portGroupUUID, portGroupName,
+	direction, priority, extraMatch, action string,
+	policyType knet.PolicyType) error {
+	var match string
+
+	if extraMatch != "" {
+		extraMatch = fmt.Sprintf(" && %s", extraMatch)
+	}
+
 	if policyType == knet.PolicyTypeIngress {
-		match = fmt.Sprintf("match=\"outport == @%s\"", portGroupName)
+		match = fmt.Sprintf("match=\"outport == @%s%s\"", portGroupName,
+			extraMatch)
 	} else {
-		match = fmt.Sprintf("match=\"inport == @%s\"", portGroupName)
+		match = fmt.Sprintf("match=\"inport == @%s%s\"", portGroupName,
+			extraMatch)
 	}
 
+	aclAction := fmt.Sprintf("action=%s", action)
```

Comment:
@dceara you could skip this and below just do:

```
"--columns=_uuid", "find", "ACL", match, "action=" + aclAction,
```

Reply:
Aside: Go lets you concat strings in the same argument with +. ovnkube has a mix of fmt.Sprintf() and +, but you only really need fmt.Sprintf() when you are using non-string arguments to make sure you get the formatting right. Anywhere that just concats strings can use +

Reply:
@dcbw Got it, thanks. I wasn't sure what's preferred.

Reply:
@dcbw There are quite a few places in `policy.go` where `fmt.Sprintf` is used for concating strings. Do we change those too? Or do we do it in a different PR?

Also, what's preferred?
`match := "ip4.src == $" + nsAddressSet + " && ip4.mcast"`
OR
`match := fmt.Sprintf("ip4.src == $%s && ip4.mcast", nsAddressSet)`

Reply:
Separate PR for changing existing instances.

Reply:
> Separate PR for changing existing instances.

Ack.